release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -2110,7 +2110,27 @@ issue by deleting the Elasticsearch deployments in sequence.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1824006[*BZ#1824006*])
 
 * {product-title} 4.4 is not shipping with a v.4.4 Metering Operator. Customers can install or continue to run the v.4.3 Metering Operator on {product-title} 4.4 clusters.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1829035[*BZ1829035*])
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1829035[*BZ#1829035*])
+
+* When updating an {product-title} cluster from version 4.3 to 4.4, the
+`cluster-etcd-operator` sometimes fails to upgrade because it is in a degraded
```

Comment:
It's the `etcd` ClusterOperator.  Might be worth making this "...the `etcd` cluster operator..."?  But folks will probably understand either way.